source/_views/post.html

```diff
@@ -42,4 +42,31 @@
             </nav>
         {% endif %}
     </article>
+
+
+{% if site.disqus.shortname and site.disqus.shortname != '' %}
+<div id="disqus_thread"></div>
+<script type="text/javascript">
+    /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
+    var disqus_shortname = '{{site.disqus.shortname}}'; // required: replace example with your forum shortname
+
+    var disqus_identifier = '{{page.disqus.identifier}}';
```

Comment:
I think we should wrap these in if blocks.

    {% if page.disqus.identifier %}var disqus_identifier = '{{page.disqus.identifier}}';{% endif %}

... I'm not sure if this is what @WyriHaximus was getting at or not? I'd just as soon not set a variable at all than set it to an empty string only because the value wasn't set in the metadata.